src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -9,16 +9,27 @@
 
 #include <cos_component.h>
 #include <mem_mgr_large.h>
-#include <cbuf.h>
+#include <cbuf_meta.h>
 #include <cbuf_mgr.h>
 #include <cos_synchronization.h>
 #include <valloc.h>
+#include <sched.h>
 #include <cos_alloc.h>
 #include <cmap.h>
 #include <cos_list.h>
 
 #include <stkmgr.h>
 
+/* debug helper */
+#define OP_NUM 10
+/* 0-create 1-collect 2-delete */
+/* 3-retrieve 4-register */
+/* 56-reserve 7-cbuf_alloc_map */
+int op_nums[OP_NUM];
+unsigned long long per_total[OP_NUM];
```

Comment:
It is not at all clear from the name that this is tracking time. I recommend making the type match with the calculations (u64_t), and adjust the name to be more clear that this is tracking time, e.g. total_tsc_per_op or something similar.